compiler/rustc_middle/src/middle/stability.rs

```diff
@@ -226,18 +226,19 @@ fn late_report_deprecation(
     suggestion: Option<Symbol>,
     lint: &'static Lint,
     span: Span,
+    method_span: Option<Span>,
```

Comment:
Maybe it's not a relevant name for now. But I'm not quite sure what to rename it.

Reply:
Same.